text_generation/causal_lm/cpp/continuous_batching/library/src/model_runner.hpp

```diff
@@ -235,11 +235,7 @@ class ModelRunner {
         m_request.set_tensor("context_lens", context_lens);
         m_request.set_tensor("block_tables", block_tables);
 
-        try {
-            m_request.set_tensor("subsequence_lens", subsequence_lens);
-        } catch (ov::Exception& ex) {
-            // ignore, because plugin's kernel does not support this new parameter
-        }
+        m_request.set_tensor("subsequence_lens", subsequence_lens);
```

Comment:
can we keep try / catch ?
- it will allow to use current code with OpenVINO master
- the same code it used for vLLM scheduling where we don't have `subsequence_lens` parameter in our model

Reply:
Restored.